code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -240,8 +240,8 @@
 
 /mob/living/simple_animal/bot/mulebot/ui_data(mob/user)
 	var/list/data = list()
-	data["on"] = get_bot_flag(BOT_MODE_ON)
-	data["locked"] = get_bot_flag(BOT_COVER_LOCKED)
+	data["on"] = bot_mode_flags & BOT_MODE_ON
+	data["locked"] = bot_cover_flags & BOT_COVER_LOCKED
```

Comment:
does the ui expect this to just be truthy? or does it expect 1/0

Reply:
I'd assume it checks if it has that flag, it's already what bots/beepsky does, and works fine.

Reply:
k